app/models/exchange_rate_currency_rate.rb

```diff
@@ -81,6 +95,15 @@ def max_year
         &.year.presence || Time.zone.today.year
     end
 
+    def max_month
```

Comment:
what about calling this `most_recent_month`,
also, silly question, where is it used?